Core/src/main/java/com/plotsquared/core/player/PlotPlayer.java

```diff
@@ -841,6 +841,20 @@ public void sendTitle(
                 .title(titleComponent, subtitleComponent, times));
     }
 
+    /**
+     * Method designed to send an ActionBar to a player.
+     *
+     * @param caption Caption
+     * @param replacements Variable replacements
+     */
+    public void sendActionBar(
+            final @NonNull Caption caption,
+            final @NonNull Template... replacements
+    ) {
+        final Component component = MiniMessage.get().parse(caption.getComponent(this), replacements);
```

Comment:
To fix the issue with <prefix> not getting replaced, this should be solved similar to how it is written in `sendMessage` below (means call `replace` on the raw String returned by `getComponent(this)`).